.eslintrc

```diff
@@ -22,7 +22,7 @@
     "import/no-extraneous-dependencies": 0,
     "import/no-unresolved": 0,
     "import/prefer-default-export": 0,
-    "indent": 2,
+    "indent": 1,
```

Comment:
Any reason why we only want to warn on indentation mistakes? Should we default to forcing correct indentation?

Reply:
The reasons it's being set as a warning is because of development speed.
Currently the applications stops showing the page and shows errors on 2. SO when you're developing and your indentation isn't correct because it's "pseudo" code, then you can't see your page.

Like this we'll know to fix but won't be limited when developing.